core/kazoo_number_manager/src/knm_maintenance.erl

```diff
@@ -37,10 +38,36 @@
 %% API
 
 %% @public
+-spec maybe_update_number_services_view(ne_binary()) -> ok.
+maybe_update_number_services_view(?MATCH_ACCOUNT_ENCODED(_)=AccountDb) ->
```

Comment:
rename this to `update_number_services_view` since we log the result.
also add a fun clause for raw accountid